data_structures/src/chain.rs

```diff
@@ -2393,6 +2393,25 @@ pub fn get_utxo_info(
 
 pub type Blockchain = BTreeMap<Epoch, Hash>;
 
+/// Node stats
+#[derive(Debug, Default, Clone, Serialize, Deserialize, PartialEq)]
+pub struct NodeStats {
+    /// Number of proposed blocks
+    pub block_proposed_count: u32,
+    /// Number of blocks included in the block chain
+    pub block_mined_count: u32,
+    /// Data Requests with elegibility
+    pub dr_with_eligibility_count: u32,
```

Comment:
This is misleading the name suggests that this is the number of different data requests that we could have participated in, but if we propose 2 commits in 2 different rounds in the same data requests it counts as 2 data requests. So it should be renamed to something like "number of times we were eligible to participate in a data request".